### PR TITLE
Replace `#[macro_use]` with normal `use`

### LINKED ISSUE #805: Replace `#[macro_use]` with normal `use`
[druid-shell/src/lib.rs#L27](https://github.com/xi-editor/druid/blob/a8d062a919868d7dca5aac885dfec9dc99f1a313/druid-shell/src/lib.rs#L27)
```rust
#[cfg(target_os = "windows")]
#[macro_use]
extern crate winapi;

#[cfg(all(target_os = "macos", not(feature = "use_gtk")))]
#[macro_use]
extern crate objc;

#[cfg(not(any(feature = "use_gtk", target_os = "linux")))]
#[macro_use]
extern crate lazy_static;

#[cfg(all(target_os = "linux", feature = "x11"))]
#[macro_use]
extern crate lazy_static;
```
This is no longer necessary with Rust2018 and should be replaced with use statements in modules that actually utilize those macros

### PR DESCRIPTION
**Pull request by totsteps**
closes #805

### CODE REVIEW

File: druid-shell/src/platform/windows/dialog.rs [hunk 1]
```diff
@@ -31,6 +31,7 @@ use winapi::um::shobjidl::*;
 use winapi::um::shobjidl_core::*;
 use winapi::um::shtypes::COMDLG_FILTERSPEC;
 use winapi::Interface;
+use winapi::DEFINE_GUID;
```
Comment (cmyr) [inline druid-shell/src/platform/windows/dialog.rs:34]: can we combine this with the preceding import?

File: druid-shell/src/platform/x11/application.rs [hunk 2]
```diff
@@ -24,6 +24,8 @@ use crate::application::AppHandler;
 use crate::kurbo::Point;
 use crate::{KeyCode, KeyModifiers, MouseButton, MouseEvent};
 
+use lazy_static::lazy_static;
```
Comment (xStrom) [inline druid-shell/src/platform/x11/application.rs:27]: Not a big deal right now, but generally we use the following order with imports:
1. std modules
2. external modules
3. internal modules

Comment (totsteps) [reply druid-shell/src/platform/x11/application.rs]: Re-ordered imports for all the files under `druid-shell`.

Though some files under `druid-derive` and `druid` don't follow this order!

Comment (xStrom) [reply druid-shell/src/platform/x11/application.rs]: Thanks a lot! :+1:

Yes it's true this is not properly followed. My comment here and your response is part of the process in getting it fixed. :slightly_smiling_face:

Comment (totsteps) [reply druid-shell/src/platform/x11/application.rs]: Sorry, I guess my comment may seem somewhat rough.
What I meant was that some files under `druid-drive` and `druid` do not follow import order and should be fixed.

I would love to fix those as well, maybe in this or other PR?

Comment (xStrom) [reply druid-shell/src/platform/x11/application.rs]: There's nothing unpleasant, no worries. Yes you can send a new PR with those changes. An addition to the contribution guide is also welcome.

This PR here should just wait for merging now.

File: druid-shell/src/keyboard.rs [hunk 3]
```diff
@@ -14,9 +14,10 @@
 
 //! Keyboard event types and helpers
 
-use super::keycodes::KeyCode;
 use std::fmt;
 
+use super::keycodes::KeyCode;
```
Comment (cmyr) [inline druid-shell/src/keyboard.rs:19]: thanks 😄

**Pull request merged**